frontend/components/surplus/SurplusAuctionBidTransactionTable.vue

```diff
@@ -0,0 +1,152 @@
+                        :disabled="!isActive"
+                        :validator="validator"
+                    />
+                </div>
+            </div>
+        </div>
+        <div class="flex justify-between">
+            <div>Combined Transaction Fees</div>
+            <div>
+                <FormatCurrency v-if="auction.combinedBidFees" :value="auction.combinedBidFees" currency="DAI" />
+                <span v-else class="opacity-50">Unknown</span>
+            </div>
+        </div>
+        <div class="flex justify-between">
+            <div>Price on Uniswap</div>
+            <div>
+                <format-currency
+                    v-if="auction.marketUnitPrice && isActive"
+                    :value="auction.marketUnitPrice"
+                    currency="MKR"
+                />
+                <span v-else class="opacity-50">Unknown</span> per DAI
+            </div>
+        </div>
+        <div class="flex justify-between font-bold">
+            <div>Price after the bid</div>
+            <div>
+                <format-currency
+                    v-if="unitPriceAfterBid && isActive && !isBidAmountNaN"
+                    :value="unitPriceAfterBid"
+                    currency="MKR"
+                />
+                <span v-else class="opacity-50">Unknown</span> per DAI
+            </div>
+        </div>
+    </div>
+</template>
+
+<script lang="ts">
+import type { SurplusAuction } from 'auctions-core/src/types';
+import Vue from 'vue';
+import BigNumber from 'bignumber.js';
+import BidInput from '../utils/BidInput.vue';
+import TimeTill from '~/components/common/TimeTill.vue';
+import FormatCurrency from '~/components/utils/FormatCurrency.vue';
+
+export default Vue.extend({
+    components: {
+        TimeTill,
+        FormatCurrency,
+        BidInput,
+    },
+    props: {
+        auction: {
+            type: Object as Vue.PropType<SurplusAuction>,
+            required: true,
+        },
+        lowestNextBid: {
+            type: Object as Vue.PropType<BigNumber>,
+            required: true,
+        },
+    },
+    data() {
+        return {
+            inputBidAmount: undefined as BigNumber | undefined,
+        };
+    },
+    computed: {
+        isActive(): boolean {
+            return this.auction.state === 'just-started' || this.auction.state === 'have-bids';
+        },
+        isBidAmountNaN(): boolean {
+            return !!this.transactionBidAmount?.isNaN();
+        },
+        unitPriceAfterBid(): BigNumber | undefined {
+            if (!this.transactionBidAmount) {
+                return this.lowestNextBid.dividedBy(this.auction.receiveAmountDAI || 1);
+            }
+            return this.transactionBidAmount.dividedBy(this.auction.receiveAmountDAI || 1);
+        },
+    },
+    watch: {
+        inputBidAmount: {
+            immediate: true,
+            handler(inputBidAmount) {
+                this.$emit('inputBidAmount', inputBidAmount);
+            },
+        },
```

Comment:
I still see `inputBidAmount` as well as `transactionBidAmount` throughout this PR. My question remains: are they the same thing? If so, should we rename `inputBidAmount` into `transactionBidAmount`?

Reply:
Yes, they are. I will rename them all to `inputBidAmount`. I assume I missed a few, but ill make sure. 
I believe `inputBidAmount` is better, as it is more general then `transactionBidAmount`.

Reply:
I actully have to rollback on this. While replacing everything I noticed, that `transactionBidAmount` is used [here](https://github.com/sidestream-tech/unified-auctions-ui/blob/main/frontend/components/transaction/BidTransaction.vue#L177) to calculate what the amount is based on the users `inputBidAmount` or the min or max values, if no input is provided. 


Therefore I suggest (and changed) all references that can be changed by the user to `inputBidAmount` to keep the terminology separated meaning:

`inputBidAmount` = the value directly changed by the user
`transactionBidAmount` = the value we calculate dependent on the user input and other values.

Sorry for all the confusion beforehand!